openquake/server/local_settings.py.tools

```diff
@@ -1,3 +1,6 @@
+import sys
```

Comment:
Is `sys` needed too?

Reply:
Fixed with this commit https://github.com/gem/oq-engine/commit/423d1b75e10d3132cfa75e4f2f46ad7908fe501c